mods/d2k/rules/arrakis.yaml

```diff
@@ -1,20 +1,19 @@
 spicebloom:
 	HiddenUnderShroud:
-	RenderBuilding:
-	Building:
-		Footprint: x
-		Dimensions: 1,1
+	RenderSprites:
 	AppearsOnRadar:
+		UseLocation: yes
+	EditorAppearance:
+		RelativeToTopLeft: yes
+	ProximityCaptor:
+		Types: Tree
```

Comment:
This is not a style problem. You chose the wrong version when resolving rebase conflicts.